handle.go

```diff
@@ -448,7 +448,11 @@ func (h HandleType) TsBatch(cols ...TsBatchColumnInfo) (*TsBatch, error) {
 
 func (h HandleType) GetLastError() (string, error) {
 	var err C.qdb_error_t
-	var message C.qdb_string_t
+	var message *C.qdb_string_t = nil
```

Comment:
I don't know a lot about go FFI, but how is memory managed here? It's now a pointer and needs to be deallocated after it's copied -- how does this work? Does C.GoString copy the memory? 

It's probably not the worst memory leak, but it would be interesting to look into.

Reply:
Good point. This is totally an unknown territory to me. I'll have to do more research about it.

Reply:
If you have time to do it, do it. There are probably hints on how to do it in other parts of the code, e.g. how are query results released?